pkg/sm/sm.go

```diff
@@ -89,6 +91,9 @@ func New(ctx context.Context, env env.Environment) *ServiceManagerBuilder {
 	// setup logging
 	ctx = log.Configure(ctx, cfg.Log)
 
+	// goroutines that log must be started after the log has been configured
+	util.HandleInterrupts(ctx, cancel)
```

Comment:
can we put it in the server for now ? otherwise we gotta do the same in proxy.go

Reply:
I agree we'd have to copy it, but in the server seems to be too late. Calls to the database, as well as spawning a new goroutine happen before the server is created. If anything fails, the storage might not be closed and goroutines will leak